### ISSUE TITLE
issue with header 

### ISSUE THREAD

**Issue by oritRosengarten** [Scripta-Qumranica-Electronica/ScrollEditor#372]
After I enter an edition the header of the same edition appears when I'm in search mode and other screens, it should clear up when I am not actually in the edition  

![Capture](https://user-images.githubusercontent.com/74494426/154916060-9b07fa97-e9d5-4079-ba09-d02d2d052603.JPG)

**Comment by zmbq** [comment]
I don't think so, it's a quick and easy way to get back to the previous edition and keep working on it. Are you sure you want us to remove this?
